.gemini.yml

```diff
@@ -1,22 +1,16 @@
-rootUrl: http://localhost:8080/components/vaadin-radio-button/test/visual/
+rootUrl: http://localhost:8080/test/visual/
 gridUrl: http://localhost:4444/wd/hub
 screenshotsDir: ./test/visual/screens
 
 system:
   plugins:
-    polyserve:
+    esm:
       port: 8080
     sauce: true
 
 browsers:
   chrome:
     desiredCapabilities:
       browserName: "chrome"
-      version: "67.0"
-      platform: "Windows 10"
-
-  firefox:
```

Comment:
just curious: why removed firefox ?

Reply:
Mainly because of gemini not supporting Firefox 64+ which had native Shadow DOM.

See https://github.com/gemini-testing/gemini/pull/948

Reply:
thank you for answering :)